compiler/lib/global_deadcode.ml

```diff
@@ -0,0 +1,445 @@
+(* Js_of_ocaml compiler
+ * http://www.ocsigen.org/js_of_ocaml/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, with linking exception;
+ * either version 2.1 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+
```

Comment:
The comment should be split.
- first header 
- then doc

Reply:
[658dcb0](https://github.com/ocsigen/js_of_ocaml/pull/1503/commits/658dcb0dcadbfc6ec7db1d4824b797e8c4ff06bc) moved this to a doc comment in the `.mli` file which I think is right but I'm not an `odoc` expert.